docs/guides/getting-started.md

```diff
@@ -0,0 +1,95 @@
+---
+page_title: Getting Started
+---
+
+# Getting Started
+
+## Installing Terraform
+
+See [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) for more information.
+
+## Declaring a Task
+
+In an empty directory:
+
+1. Create a directory named `shared` to store input data and output artifacts.
```

Comment:
- Does it have to be `shared/` or is it an example?
- So you create an empty dir (any name) and then another dir inside called `shared/` ?

Reply:
May be clearer to show a simple `tree` or consolidate these steps into a simple sentence like "Create a `shared/main.tf` config file inside your project with these contents:"

Reply:
> _Does it have to be `shared/` or is it an example?_

It's an example, matching the code snippet below.

> _So you create an empty dir (any name) and then another dir inside called `shared/` ?_

Yes, technically the nested directory can also have an arbitrary name, but we have to choose one to mention it on the code snippet.

Reply:
> _May be clearer to show a simple tree or consolidate these steps into a simple sentence like "Create a shared/main.tf config file inside your project with these contents:"_

The `main.tf` file should be outside of the `shared` directory. That's the point of creating a directory just for transferring data.

```
project/
├─ shared/
│  └─ ···
└─ main.tf
```

Reply:
The tree is clearer 👍 

`project/` = `directory/` in the samples? In general I'd just try to make the samples match the text as clearly as possible. Thanks

Reply:
No, `project/shared/` is the value of the `directory` argument in the samples. There is no (or there shoulnd't be, at least) such thing as `directory/` in the samples.

Reply:
If you wish, we can remove the `path.root` name just to keep things simple.
````
.
├─ shared/
│  └─ ···
└─ main.tf
```

Reply:
Committed